src/bucket.js

```diff
@@ -1281,6 +1289,10 @@ Bucket.prototype.getFiles = function(query, callback) {
           options.generation = file.generation;
         }
 
+        if (file.kmsKeyName) {
+          options.kmsKeyName = file.kmsKeyName;
```

Comment:
Should we be checking that we're not overwriting an already specified `kmsKeyName`?

Reply:
Just a guess, but the diff in this PR makes it look like we have an option `[options.kmsKeyName] The name of the...` to this method, but that's actually for a separate method, `bucket.file()`.

But if that's not what maybe got you mixed up, there can be a default KMS key name on a Bucket, (set via `setMetadata`), but if a File has its own KMS key name, I believe we need to use that one.

LMK if I misunderstood!

Reply:
Want to make sure this is resolved.

Reply:
Yes, I think this was just confusion on how the PR diff was displaying the changes. We use the object's KMS key name that comes back from the API.